conda/petsc/build.sh

```diff
@@ -44,29 +47,32 @@ BUILD_CONFIG=`cat <<"EOF"
   --with-cxx-dialect=C++11 \
   --with-shared-libraries=1 \
   --download-mumps=1 \
-  --download-strumpack=1 \
   --download-hypre=1 \
   --download-metis=1 \
   --download-slepc=1 \
   --download-ptscotch=1 \
   --download-parmetis=1 \
+  --download-strumpack=1 \
   --download-scalapack=1 \
+  --download-fblaslapack=1 \
```

Comment:
Should we just call "update_and_rebuild_petsc.sh" from here?

Reply:
If possible - I would really prefer this. It would be nice if we could add the additional conda require things to `update_and_rebuild_petsc.sh` so that we can for sure keep them in sync as much as possible

Reply:
It's a good idea. I'll look into it!

Reply:
@fdkong, I think we've addressed this with the combo of a5e3b39 and 334ca0c if you want to take a look.